tests/test_unit_operation.py

```diff
@@ -60,6 +64,18 @@ def create_tubular_reactor(self):
 
         return tube
 
+    def create_MCT(self):
+        mct = MCT(ComponentSystem(1), nchannel=3, name='test')
+
+        mct.length = length
```

Comment:
We should consider moving this outside the `Test_Unit_Operation` class definition s.t. we could potentially reuse the same fixture for other tests.